### PR TITLE
docs: update

### LINKED ISSUE #460: Weave docs need some updating
In following the installation and configuration instructions in the [Weave](https://weave.readthedocs.io/en/latest/index.html) docs I found a few items that need correcting:

Start the [blockchain ](https://weave.readthedocs.io/en/latest/mycoind/installation.html
)section 
- [x] Replace the command `tendermint node --home ~/.mycoind --p2p.skip_upnp > ~/.mycoind/tendermint.log &` with `tendermint node --home ~/.mycoind --p2p.upnp --proxy_app 'noop'> ~/.mycoind/tendermint.log &`
- [x] Add a note : For help and explanations for the tendermint node commands `: tendermint node --help`

Config [section](https://weave.readthedocs.io/en/latest/deployment/configuration.html#application_config
)
- [x] Link to Tendermint Docs is broken: Should be https://tendermint.com/docs/introduction/
- [x] Replace `Dev: - --p2p.skip_upnp: Don’t try to determine external address.` with `Dev: - --p2p.upnp --proxy_app ‘noop’: Don’t try to determine external address (‘noop’ for local testing)`

### PR DESCRIPTION
**Pull request by husio**
fix #460

### CODE REVIEW

File: docs/deployment/configuration.rst [hunk 1]
```diff
@@ -66,8 +66,8 @@ these should be writen to the ``config.toml`` file or stored in
 environmental options in the service ini (if using 12-factor style).
 
 Dev:
-- ``--p2p.skip_upnp``: Don't try to determine external address.
-  Needed if running without internet (eg. in an airplane)
+- ``--p2p.upnp --proxy_app noop``: Don’t try to determine external address
```
Comment (kmw101) [inline docs/deployment/configuration.rst]: The command needs noop in single quotes ``--p2p.upnp --proxy_app 'noop'``: Don’t try to determine external address

Comment (ethanfrey) [reply docs/deployment/configuration.rst]: Why need the --proxy_app line at all for the skip upnp stuff?

I think default is now to not attempt, and `--p2p.upnp` enables it (or `--p2p.upnp=false` to explicitly disable if enabled via config of environment.

(I spent a lot of time with cobra, the flag parser they use)

**Comment by husio** [comment]
> In the config the command --proxy_app needs to have 'noop' in single quotes

That is weird. Shell should pass it to the program as `noop` not `'noop'` anyway. Does it really make a difference?

**Comment by kmw101** [comment]
> > In the config the command --proxy_app needs to have 'noop' in single quotes
> 
> That is weird. Shell should pass it to the program as `noop` not `'noop'` anyway. Does it really make a difference?

No difference I guess, I copied what worked when I tested it.

File: docs/deployment/configuration.rst (hunk 1 above)
Comment (kmw101) [reply docs/deployment/configuration.rst]: I added --proxy_app through trial and error when having a go at starting tendermint. This combo worked for me.

**Pull request merged**